src/natcap/invest/validation.py

```diff
@@ -15,60 +15,15 @@
 from osgeo import osr
 
 from . import gettext
+from . import spec
```

Comment:
I couldn't see where `spec` was being used in any of the additions. Is this import needed?